### PR TITLE
Add DBIS refrence #1864

### PR DESCRIPTION
**Pull request by TobiasNx**
See #1864

### CODE REVIEW

File: src/test/resources/alma-fix/990133067580206441.json [hunk 1]
```diff
@@ -79,8 +79,8 @@
     "id" : "http://ld.zdb-services.de/resource/2163340-X",
     "label" : "ZDB-Ressource"
   }, {
-    "id" : "https://d-nb.info/026537966",
-    "label" : "DNB-Ressource"
+    "id" : "http://dbis.uni-regensburg.de/frontdoor.php?titel_id=1815",
+    "label" : "DBIS-Ressource"
   }, {
```
Comment (dr0i) [inline src/test/resources/alma-fix/990133067580206441.json:84]: In https://github.com/hbz/lobid-resources/issues/1864#issuecomment-2296603227 it's said to _add_ a link to DBIS, not to drop the link to dnb.

**Review by dr0i — APPROVED** [review]
Besides of my inline remark: +1

File: src/test/resources/alma-fix/990133067580206441.json (hunk 1 above)
Comment (TobiasNx) [reply src/test/resources/alma-fix/990133067580206441.json:84]: This is weird

**Comment by TobiasNx** [comment]
This was an error in the mapping.

**Review by dr0i — APPROVED** [review]

**Comment by dr0i** [comment]
Will be deployed next Monday.

**Pull request merged**